### ISSUE TITLE
`mini.indentscope` doesn't play well with `showbreak` option

### ISSUE THREAD

**Issue by simonmandlik** [echasnovski/mini.nvim#1130]
### Contributing guidelines

- [X] I have read [CONTRIBUTING.md](https://github.com/echasnovski/mini.nvim/blob/main/CONTRIBUTING.md)
- [X] I have read [CODE_OF_CONDUCT.md](https://github.com/echasnovski/mini.nvim/blob/main/CODE_OF_CONDUCT.md)
- [X] I have updated 'mini.nvim' to latest version

### Module(s)

`mini.indentscope`

### Description

Hi, I'm using `mini.indentscope` together with `showbreak`. The indentscope drawing covers the `showbreak` characters even when the priority is set to 0:

https://github.com/user-attachments/assets/201f6712-8589-4356-99ec-ce399c0b816a

When I set the priority lower than 0, indentscope is not shown at all (even in cases when no showbreak characters are displayed)

### Neovim version

0.10.1

### Steps to reproduce

```lua
vim.env.LAZY_STDPATH = ".repro"
load(vim.fn.system("curl -s https://raw.githubusercontent.com/folke/lazy.nvim/main/bootstrap.lua"))()

require("lazy.minit").repro({
    spec = {
        { "folke/flash.nvim", opts = {} },
        -- add any other plugins here
        {
            "echasnovski/mini.indentscope",
            config = function()
                require("mini.indentscope").setup {
                    draw = {
                        delay = 0,
                        priority = 0,
                        -- priority = -1
                    },
                }
            end
        }
    }
})

vim.opt.showbreak = "x"
vim.opt.breakindent = true
vim.opt.breakindentopt:append "sbr"
```

**Comment by echasnovski** [comment]
Thanks for the issue!

This is the case after resolving #635 and seems to an easy fix (similar to what resolved #810, i.e. 57f98fe).

**Comment by echasnovski** [comment]
This should now be resolved on latest `main`.
